packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -618,6 +627,25 @@ class StackViewport extends Viewport implements IStackViewport {
     return imagePlaneModule;
   }
 
+  /**
+   * Sets the default properties for the viewport
+   * @param ViewportProperties
+   * @param imageIdIndex If given, we st the default properties only for this image index
+   */
```

Comment:
this needs more documentation on how first time a properties is set it will set the defaultProperties for all imageIds too, and this is basically for modifying that